src/Diagnostics.HealthChecks.AspNetCore/Microsoft.Omex.Extensions.Diagnostics.HealthChecks.AspNetCore.csproj

```diff
@@ -12,9 +12,9 @@
     <PackageTags>Microsoft;Omex;Extensions;HealthChecks;AspNetCore</PackageTags>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" />
+    <ProjectReference Include="..\Diagnostics.HealthChecks\Microsoft.Omex.Extensions.Diagnostics.HealthChecks.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Diagnostics.HealthChecks\Microsoft.Omex.Extensions.Diagnostics.HealthChecks.csproj" />
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
The `FrameworkReference` section should include all the AspNetCore dependencies normally contained in the `Microsoft.AspNetCore.App.Ref` package that in turn contains the reference to the [ActionFilterAttribute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.filters.actionfilterattribute?view=aspnetcore-7.0) abstract class.